### PR TITLE
1.27.2 Hotfix doc changes

### PR DESCRIPTION
**Pull request by FlappiTomic**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

**Review by jackjia-ibm — APPROVED** [review]

**Review by t1m0thyj — APPROVED** [review]

File: docs/getting-started/release-notes/v1_27_2.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Welcome to the Version 1.27.2 release of Zowe!
+
+**Download v1.27.2 build:** Want to try v1.27.2 as soon as possible? You can download the v1.27.2 build from [Zowe.org](https://www.zowe.org/download.html).
+
+## Bug fixes
+
+### Zowe Application Framework
+#### zLUX Server Framework
+
+* Bugfix: keyring_js did not worked properly for finding CAs due to using an older version in package.json than needed for the listKeyring function. [(#766)](https://github.com/zowe/zlux/issues/766). The fix is in [(PR-384)](https://github.com/zowe/zlux-server-framework/pull/384)
```
Comment (t1m0thyj) [inline docs/getting-started/release-notes/v1_27_2.md:12]: `did not worked` -> `did not work`

File: docs/getting-started/release-notes/v1_27_2.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+The following bugs were fixed in **core Zowe CLI**:
+
+* BugFix: Updated log4js and nanoid for improved security.
+
+#### Imperative CLI Framework
+
+The following bugs were fixed in the **Imperative CLI Framework**:
+
+* BugFix: Fixed web help wrongly escaping characters inside code blocks. [(#730)](https://github.com/zowe/imperative/issues/730)
+* BugFix: Fixes an error where, in certain situations, the web help displays data for another command with the same name. [(#728)](https://github.com/zowe/imperative/issues/728)
```
Comment (t1m0thyj) [inline docs/getting-started/release-notes/v1_27_2.md:25]: `Fixes` -> `Fixed` - for consistency I believe we prefer past tense

**Review by t1m0thyj — CHANGES_REQUESTED** [review]
Sorry, caught a typo that I missed earlier

**Comment by FlappiTomic** [comment]
> Sorry, caught a typo that I missed earlier

Good catch! I delivered the typo fixes.

**Review by t1m0thyj — APPROVED** [review]

**Pull request merged**